asim: TestAllocatorSimulatorSpeed flaky

```
26.381840782
    asim_test.go:159:
          Error Trace:  /go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/asim/asim_test.go:159
          Error:        "26381840782" is not less than "25000000000"
          Test:         TestAllocatorSimulatorSpeed
--- FAIL: TestAllocatorSimulatorSpeed (156.40s)
```

https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests/6716561?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true

--- a/pkg/kv/kvserver/asim/asim_test.go
+++ b/pkg/kv/kvserver/asim/asim_test.go
@@ -142,7 +142,7 @@ func TestAllocatorSimulatorSpeed(t *testing.T) {
 	minRunTime := int64(math.MaxInt64)
 	// TODO(kvoli): Hit perf wall again when running at a lower tick rate on
 	// selected operartions.
-	requiredRunTime := 25 * time.Second.Nanoseconds()
+	requiredRunTime := 30 * time.Second.Nanoseconds()
 	samples := 5
 	for i := 0; i < samples; i++ {
 		if sampledRun := sample(); sampledRun < minRunTime {

